frontend/src/components/dashboard/create/sms/SMSRecipients.tsx

```diff
@@ -127,9 +127,8 @@ const SMSRecipients = ({
       setCsvInfo((info) => ({ ...info, tempCsvFilename }))
     } catch (err) {
       setErrorMessage(err.message)
```

Comment:
Should we also check if the component is still mounted before choosing to call `setErrorMessage`?

Reply:
Possibly, but I haven't noticed a real issue with it so far. That said, I might run into the error when unit testing error states for this component. I'll try it out and let you know how it goes.

Reply:
Sounds fair.